contents/handbook/growth/sales/sales-operations.md

```diff
@@ -6,120 +6,66 @@ showTitle: true
+6. We track implementation in a [GitHub project](https://github.com/orgs/PostHog/projects/10). The first 1-2 months are spent scaling the instance properly so we don't go too big and waste customers' money. 
+7. Once the customer is ready to begin event ingestion, this is the point at which we will ask for payment details, so we can start tracking usage. Paolo will generate a payment link in Stripe. 
+8. The final part of initial setup is to schedule a call to help them set up their first dashboards and ensure they are getting the most out of PostHog. Ongoing support is provided in the shared Slack channel. 
 
-Hubspot is a comprehensive tool with a lot of functionality, so we are currently focused on using a few core features well. You are most likely to use the following regularly:
+### Figuring out the best solution for a customer
 
-- _Contacts_ - pretty straightforward, under 'Contacts'. You can create contacts manually, or sync with your Gmail. 
-- _Companies_ - also under 'Contacts'. You will also want to create a company record to associate with any contact (and you can associate multiple contacts with a single company). If you enter the company's domain name, HubSpot is pretty good at pulling in additional data to fill out the record. 
-- _Inbox_ - this is under 'Conversations' and is where we deal with messages that come into our public-facing email addresses. New messages will come in as 'Unassigned' and then get assigned to someone.
-- _Deals_ - under 'Sales'. This is where we manage our customers who are interested in an Enterprise or Startup plan and is the core of our sales ops process. 
-- _Tasks_ - also under 'Sales'. This is a useful place to see a summary of all the tasks that you have created or that have been assigned to you. 
+Assuming PostHog is the best solution for a customer, you should look at their level of scale and if they have any specific privacy or security needs to determine the most appropriate plan for them.  
 
-If you'd like to dig deeper, HubSpot have a ton of [documentation](https://knowledge.hubspot.com/) and resources that you can refer to as well.
+- _Low volume, less technical_ - start with Cloud, which is free up to 1m monthly events and very fast to get going with. 
+- _Low volume, more technical_ - Open Source will be fine up to 10k MAUs. Beyond that, the product will still work but Postgres limitations at scale means performance will be degraded. 
+- _High volume, less technical_ - Cloud will be the best bet - pricing does increase at scale as we take on hosting costs, but the setup process and ongoing maintenance is very straightforward. 
+- _High volume, more technical_ - either Scale or Cloud. Scale will be more appealing for customers who need to keep control of their data for privacy or corporate policy reasons. The pricing for Scale is greatly discounted at higher volumes vs. Cloud as we don't pay hosting costs. 
```

Comment:
Scale is appropriate here, not Cloud... since it's less expensive with high volume. The only time cloud makes sense for technical high volume people is if they want absolutely zero hassle, don't care about privacy and aren't budget focussed